### PR TITLE
LTC fix DH keygen xsize

### PR DESCRIPTION
**Pull request by jforissier**

### CODE REVIEW

File: core/lib/libtomcrypt/src/pk/dh/dh_make_key.c [hunk 1]
```diff
@@ -48,14 +48,12 @@
 
 int dh_make_key(prng_state *prng, int wprng, void *q, int xbits, dh_key *key)
 {
-	const int limit = 500;	/* number of tries */
-	int err, i;
+	int err;
```
Comment (etienne-lms) [inline core/lib/libtomcrypt/src/pk/dh/dh_make_key.c:51]: `= 0;`

**Review by etienne-lms — COMMENTED** [review]
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>` with a minor comment.

**Comment by jenswi-linaro** [comment]
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

File: core/lib/libtomcrypt/src/pk/dh/dh_make_key.c (hunk 1 above)
Comment (jforissier) [reply core/lib/libtomcrypt/src/pk/dh/dh_make_key.c:51]: ok

**Pull request merged**